tests/test_cli.py

```diff
@@ -353,7 +367,188 @@ def test_mapping_to_jinja():
 
     result = runner.invoke(
         mapping_to_jinja_cli,
-        ['-', '-', '--type', type_name],
+        ['-', '-', '--es_type', type_name],
+        input=json.dumps(mapping),
+    )
+
+    assert_no_exception(result)
+    # use the child template to modify the output template.
+    jinja_env = jinja2.Environment(
+        loader=jinja2.DictLoader({'generated': result.output}))
+    gen_output = json.loads(jinja_env.from_string(child_template).render())
+    assert gen_output == expected
+
+
+def test_multiple_mapping_to_jinja():
+    """Test mapping_to_jinja."""
+    mapping = {
+        "mappings": {
+            "test1": {
+                "properties": {
+                    'sub1': {
+                        'type1': 'string',
+                    }
+                }
+            },
+            "test2": {
+                "properties": {
+                    'sub2': {
+                        'type2': 'string',
+                    }
+                }
+            }
+        }
+    }
+
+    # template which extends the generated one and will be compiled.
+    child_template = """
+    {% extends "generated"%}
+    """
+
+    # build expected output
+    expected = copy.deepcopy(mapping)
+
+    runner = CliRunner()
+
+    result = runner.invoke(
+        mapping_to_jinja_cli,
+        ['-', '-'],
+        input=json.dumps(mapping),
+    )
+
+    assert_no_exception(result)
+    # use the child template to modify the output template.
+    jinja_env = jinja2.Environment(
+        loader=jinja2.DictLoader({'generated': result.output}))
+    content = jinja_env.from_string(child_template).render()
+    gen_output = json.loads(content)
+    assert gen_output == expected
+
+
+def test_mapping_to_jinja_iconsistencies():
+    """Test mapping_to_jinja."""
```

Comment:
A comment describing what the test tries to detect would be great. If I understand correctly the test, something like this would be appropriate:
```
"""Test mapping_to_jinja.

   Check that mapping_to_jinja fails if --es_type is set when it is called on a mapping having multiple types in it.
"""
```